index.html

```diff
@@ -25,6 +25,7 @@
                 <li><a href="card.html">CARD</a></li>
                 <li><a href="Images.Html">IMAGE</a></li>
                 <li><a href="Input.html">INPUT</a></li>
+                <li><a href="Text-Utilies.html">Text-Utilies</a></li>
```

Comment:
correct the spelling of "utilities"

Reply:
typing error will take care from next time